vf_convenience/client.lua

```diff
@@ -0,0 +1,209 @@
+local objects = {"p_poly_bag_01_s", "prop_till_01", "prop_till_02", "p_till_01_s"}
+local modeltypes = {'prop_till_01', 'prop_till_02', 'p_till_01_s'}
+
+function DisplayHelpLabel(label, sublabel, time)
+  ClearBrief()
+  
+  BeginTextCommandDisplayHelp(label)
+  if sublabel then
+     AddTextComponentSubstringPlayerName(sublabel)
+  end
+
+  if time then
+    displayTime = time
+  else
+    displayTime = 10000
+  end
+  EndTextCommandDisplayHelp(0, 0, false, displayTime)
+end
+
+Citizen.CreateThread(function()
+  ClearPedBloodDamage(PlayerPedId())
+  ClearPlayerWantedLevel(PlayerId())
+
+  for k,v in pairs(objects) do
+    local object = GetHashKey(v)
+    RequestModel(object)
+    while not HasModelLoaded(object) do
+      Wait(10)
+    end
+  end
+
+  RequestAnimDict("mp_am_hold_up")
+  while not HasAnimDictLoaded("mp_am_hold_up") do
+    Wait(10)
+  end
+
+  for _, item in pairs(locations) do
+    local bCoords = item["blip"]
+    local currentBlip = AddBlipForCoord(bCoords.x, bCoords.y, bCoords.z)
+
+    SetBlipSprite(currentBlip, 52)
+    SetBlipScale(currentBlip, 0.7)
+    SetBlipAsShortRange(currentBlip, true)
+    SetBlipNameFromTextFile(currentBlip, "collision_78czu4")
+  end
+
+  while true do
+    Wait(700)
+    IsNearStore = isNearGeneralStore()
+    if IsNearStore then
+      IsStoreKeeperAlive = DoesEntityExist(storeKeeper)
+    end
+    PlayerPed = PlayerPedId()
+    PedCoords = GetEntityCoords(PlayerPed, true)
+
+    if GetClockHours() == 07 then
+      speech = "SHOP_GREET_START"
+    elseif GetClockHours() >= 09 and GetClockHours() <= 22 then 
+      speech = "SHOP_GREET"
+    else
+      speech = "SHOP_GREET_END"
+    end
+  end
+end)
+
+Citizen.CreateThread(function()
+  DestroyAllCams(0)
+  Wait(5000)
+
+  while true do
+    Wait(10)
+      if IsNearStore then
+        if IsStoreKeeperAlive then
+          if IsPlayerNearShopKeeper(storeKeeper) and not IsPedDeadOrDying(storeKeeper) then
```

Comment:
Minor suggestion but these could be fused together to 1 if statement unless I'm missing something.